woocommerce/class-sv-wc-plugin-compatibility.php

```diff
@@ -491,7 +491,17 @@ protected static function is_wc_subscriptions_version( $version ) {
 		return $version === $subscriptions_version || ( $subscriptions_version && version_compare( $version, $subscriptions_version, '=' ) );
 	}
 
+	/**
+	 * Whether the HPOS is enabled.
```

Comment:
```suggestion
	 * Determines whether the HPOS is enabled.
```

Prefer descriptive titles indicating what the method does in third person